pkg/engine/vulnerability_utils.go

```diff
@@ -129,6 +129,30 @@ func getStringFromMap(vulnParam, defaultParam, overrideKey string, vObj map[stri
 	}
 	return *ts
 }
+func getBoolFromMap(
```

Comment:
this method is triggering unnecessary errors like: `Saving result. failed to detect experimental` when trying to scan a project using **"no experimental" queries**.

Reply:
done